PIL/Image.py

```diff
@@ -504,26 +535,30 @@ class Image(object):
     _close_exclusive_fp_after_loading = True
 
     def __init__(self):
+        # type: () -> None
         # FIXME: take "new" parameters / other image?
         # FIXME: turn mode and size into delegating properties?
         self.im = None
         self.mode = ""
         self.size = (0, 0)
-        self.palette = None
-        self.info = {}
+        self.palette = None  # type: Optional[ImagePalette.ImagePalette]
+        self.info = {}  # type: Dict[Text, Any]
```

Comment:
We can certainly work on this, though I'm not sure exactly what it might represent. If the set of keys/values are limited, a `TypedDict` might be plausible in the future, even :)